### PR TITLE
Ajout de la nav

### PR DESCRIPTION
**Pull request by xsenikx**
Ajout de la Nav vérification S'il te plait

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -27,6 +27,16 @@
...
           </ul>
         </nav>
+        <h3 class="small">
+          Trouvez votre hébergement pour des vacances de rêve
+        </h3>
+        <p>
+          En plein centre ou en pleine nature
+        </p>
+        <form action="#">
+          <input type="text" name="location" value="Marseille,France">
```
Comment (Tutt-J) [inline index.html]: Missing  method

File: index.html [hunk 2]
```diff
@@ -27,6 +27,16 @@
...
+          <button>Rechercher</button>
```
Comment (Tutt-J) [inline index.html]: Must be input type submit

File: index.html (hunk 1 above)
Comment (Tutt-J) [inline index.html]: Missing label

File: index.html [hunk 3]
```diff
@@ -27,6 +27,16 @@
             </li>
           </ul>
         </nav>
+        <h3 class="small">
```
Comment (Tutt-J) [inline index.html]: Why h3 ? No class at this moment...

**Review by Tutt-J — COMMENTED** [review]
View my comments and put it out of header in a section with id seach_bar for example

File: index.html [hunk 4]
```diff
@@ -27,6 +27,18 @@
...
           </ul>
         </nav>
+        <h3>
+          Trouvez votre hébergement pour des vacances de rêve
+        </h3>
+        <p>
+          En plein centre ou en pleine nature
+        </p>
+        <form action="#">
+          <label for="Localistion">
```
Comment (Tutt-J) [inline index.html]: localisAtion for and name

File: index.html [hunk 5]
```diff
@@ -27,6 +27,18 @@
             </li>
           </ul>
         </nav>
+        <h3>
+          Trouvez votre hébergement pour des vacances de rêve
+        </h3>
+        <p>
+          En plein centre ou en pleine nature
+        </p>
+        <form action="#">
```
Comment (Tutt-J) [inline index.html]: missing method

File: index.html [hunk 6]
```diff
@@ -27,6 +27,18 @@
...
+          <label for="Localistion">
+          <input type="submit" name="Localistion" value="Marseille,France">
```
Comment (Tutt-J) [inline index.html]: Missing id="localisation"

File: index.html [hunk 7]
```diff
@@ -27,6 +27,18 @@
             </li>
           </ul>
         </nav>
+        <h3>
```
Comment (Tutt-J) [inline index.html]: Must be h1 or h2

File: index.html [hunk 8]
```diff
@@ -27,6 +27,18 @@
...
+        </h3>
+        <p>
+          En plein centre ou en pleine nature
+        </p>
+        <form action="#">
+          <label for="Localistion">
+          <input type="submit" name="Localistion" value="Marseille,France">
+          <button>Rechercher</button>
+          </label>
+        </form>
```
Comment (Tutt-J) [inline index.html]: Must be inside a section with id="search_bar"

**Review by Tutt-J — CHANGES_REQUESTED** [review]

**Comment by xsenikx** [comment]
Fermeture de la pull request et réouverture sur la nouvelle branch

**Pull request closed without merging**